be/src/exec/vectorized/jdbc_scanner.cpp

```diff
@@ -28,11 +28,16 @@ namespace starrocks::vectorized {
 
 JDBCScanner::~JDBCScanner() {}
 
-Status JDBCScanner::open(RuntimeState* state) {
+Status JDBCScanner::reset_jni_env() {
```

Comment:
@silverbullet233 It's because we cached `_jni_env`, and used this variable in different threads.